CMakeLists.txt

```diff
@@ -1302,6 +1302,9 @@ check_symbol_exists(getrlimit      "${CURL_INCLUDES}" HAVE_GETRLIMIT)
 check_symbol_exists(setlocale      "${CURL_INCLUDES}" HAVE_SETLOCALE)
 check_symbol_exists(setmode        "${CURL_INCLUDES}" HAVE_SETMODE)
 check_symbol_exists(setrlimit      "${CURL_INCLUDES}" HAVE_SETRLIMIT)
+check_symbol_exists(bzero          "${CURL_INCLUDES};strings.h" HAVE_BZERO)
+check_symbol_exists(explicit_bzero "${CURL_INCLUDES};strings.h" HAVE_EXPLICIT_BZERO)
```

Comment:
This page:
https://man.archlinux.org/man/explicit_bzero.3.en
suggests that the header is `string.h` (without the `s`).
(this is also what libssh2 uses.)

Reply:
Meh, of course, that's a copy-pasteo